coinlib_flutter/example/pubspec.yaml

```diff
@@ -2,15 +2,15 @@ name: coinlib_flutter_example
 description: Demonstrates how to use the coinlib_flutter plugin.
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
-version: 1.0.0
+version: 2.1.0
```

Comment:
How come this is changed to 2.1.0?

Reply:
I chose 2.1.0 because I assumed the example app would be bumped to 2.0 for v2.0, a 2.1 because this is 2.0 + windows (the example windows CMakeLists are altered).  It can be changed to anything, though, what would you prefer?  No change?

Reply:
I've left it as it is as it's only an example app. It doesn't matter what the version is really.

Reply:
And, there is possibility of the Windows support being added to v2.0. It depends on when a final release is made. I'm still waiting on feedback for the Taproot changes.

Reply:
Well please feel free to change it to whatever needed.  I'm more than happy to change it and push before merge, just let me know!

Reply:
I'd say keep it as 1.0.0 though it's not very important.